fitwsarah_frontend/src/views/AdminPanelPage/UpdateService.js

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+import { Container, Form, Button } from 'react-bootstrap';
+import "../../css/style.css";
+import NavNotLoggedIn from "../../components/navigation/NotLoggedIn/navNotLoggedIn";
+import NavLoggedIn from "../../components/navigation/loggedIn/navLoggedIn";
+import FooterNotLoggedIn from "../../components/footer/footerNotLoggedIn/footerNotLoggedIn";
+import configData from "../../config.json";
+
+function UpdateFitnessPackage() {
+    const { isAuthenticated, getAccessTokenSilently, user } = useAuth0();
+    const [accessToken, setAccessToken] = useState(null);
+    const [fitnessPackage, setFitnessPackage] = useState({
+        title: '',
+        duration: '',
+        price: '',
+        description: '',
+        otherInformation: ''
+    });
+
+    useEffect(() => {
+        if (isAuthenticated) {
+            const getAccessToken = async () => {
+                try {
+                    const token = await getAccessTokenSilently({
+                        audience: process.env.REACT_APP_AUTH0_AUDIENCE,
+                        scope: configData.scope,
+                    });
+                    setAccessToken(token);
+                } catch (e) {
+                    console.error(e.message);
+                }
+            };
+            getAccessToken();
+        }
+    }, [getAccessTokenSilently, isAuthenticated]);
+
+    const handleInputChange = (e) => {
+        const { name, value } = e.target;
+        setFitnessPackage(prevState => ({
+            ...prevState,
+            [name]: value
+        }));
+    };
+
+    const handleUpdate = (serviceId) => {
+         // Replace with actual service ID
+        fetch(`http://localhost:8080/api/v1/fitnessPackages/${serviceId}`, {
```

Comment:
should this not use BASE URL for the deployed website?

Reply:
I ended up not using this file.